scripts/docs-content-link-rewrites/helpers/get-paths-from-nav-data.ts

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+import type { fileContentEntry } from "./fetch-nav-data-for-repo-and-base-path"
```

Comment:
Rather than importing the type into this file, why don't we make a type.ts file for the scripts directory?